docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/block/section_nav/block--cgov-section-nav.html.twig

```diff
@@ -1,3 +1,5 @@
+{% set nav_tree = content['#nav_tree']%}
```

Comment:
Modifications for this file are related to adding the section root as a cache tag.